app/controllers/concerns/api/current/sync_encounter_observation.rb

```diff
@@ -0,0 +1,60 @@
+module Api::Current::SyncEncounterObservation
+  extend ActiveSupport::Concern
+  included do
+
+    def merge_encounter_observation(key, params)
+      ActiveRecord::Base.transaction do
+        if FeatureToggle.enabled?('SYNC_ENCOUNTERS')
+          # this will always return a single blood_sugar
+          add_encounter_and_merge_record(key, params)[:observations][key][0]
```

Comment:
This is a relic from before, but maybe now `add_encounter_and_merge_record` can always return an encounter with a single observation so we don't have to do this `[0]` business? What do you think?